### PR TITLE
update airflow docs links

### PR DESCRIPTION
**Pull request by leahecole**
## Description

This is a continuation of #6276 which was raised by an external contributor - thank you @sjsyz !

## Checklist
- [x] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md)
- [x] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#readme-file)
- [ ] **Tests** pass:   `nox -s py-3.6` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [x] Please **merge** this PR for me once it is approved.
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/.github/CODEOWNERS) with the codeowners for this sample

### CODE REVIEW

File: composer/airflow_1_samples/bq_copy_across_locations.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
 Lastly, the DAG defines a specific application logger to generate logs.
 
 This DAG relies on three Airflow variables
-(https://airflow.apache.org/concepts.html#variables):
+(https://airflow.apache.org/docs/apache-airflow/stable/concepts/variables.html):
```
Comment (dandhlee) [inline composer/airflow_1_samples/bq_copy_across_locations.py:22]: nit: some links are surrounded by parenthesis `(https://...)` and some are not :C

**Review by dandhlee — APPROVED** [review]
LGTM as is! Very minor nit 🤪

File: composer/airflow_1_samples/bq_copy_across_locations.py (hunk 1 above)
Comment (leahecole) [reply composer/airflow_1_samples/bq_copy_across_locations.py:22]: lol I have no idea why that one has parens. I just did a find and replace but I'll get rid of the ones in this sample

Comment (dandhlee) [reply composer/airflow_1_samples/bq_copy_across_locations.py]: Thank you!! I can sleep in peace 😴🌙

**Comment by leahecole** [comment]
Rachael is OOO so a second Python samples reviewer should be sufficient given this is docs only

**Review by kurtisvg — APPROVED** [review]

**Pull request merged**